[2022-12-12 18:40:17,360] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T17:58:00+00:00 [queued]>
[2022-12-12 18:40:17,406] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T17:58:00+00:00 [queued]>
[2022-12-12 18:40:17,411] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-12 18:40:17,412] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-12 18:40:17,414] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-12 18:40:17,459] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_last_id> on 2022-11-28 17:58:00+00:00
[2022-12-12 18:40:17,494] {standard_task_runner.py:52} INFO - Started process 683 to run task
[2022-12-12 18:40:17,523] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'prepare_data', 'get_last_id', 'scheduled__2022-11-28T17:58:00+00:00', '--job-id', '235', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmpgojrauer', '--error-file', '/tmp/tmpnc8f9e8a']
[2022-12-12 18:40:17,529] {standard_task_runner.py:80} INFO - Job 235: Subtask get_last_id
[2022-12-12 18:40:17,745] {task_command.py:369} INFO - Running <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T17:58:00+00:00 [running]> on host cfea470654c7
[2022-12-12 18:40:17,985] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=prepare_data
AIRFLOW_CTX_TASK_ID=get_last_id
AIRFLOW_CTX_EXECUTION_DATE=2022-11-28T17:58:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-28T17:58:00+00:00
[2022-12-12 18:40:18,027] {base.py:68} INFO - Using connection ID 'postgres_default' for task execution.
[2022-12-12 18:40:18,119] {python.py:173} INFO - Done. Returned value was: None
[2022-12-12 18:40:18,156] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_data, task_id=get_last_id, execution_date=20221128T175800, start_date=20221212T184017, end_date=20221212T184018
[2022-12-12 18:40:18,231] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-12 18:40:18,312] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
